adoc/deployment.adoc

```diff
@@ -50,20 +50,23 @@ The following procedure describes how to install {productnameshort} using {helm}
 Replace `<RELEASE_NAME>` with your custom release name for the {helm} chart deployment.
 
 . Log in to {sregistry} using the obtained {scca} mirroring credentials.
+[source]
+  $ head -1 ./password.txt | helm registry login registry.suse.com \
+  --username <SUSE_REGISTRY_USERNAME> --password-stdin
 . Install the latest version of the {productnameshort} {helm} chart.
   To override the default installation with custom values from the `suse_registry_override.yaml` file`, refer to <<pr-helm-chart-override>>.
```

Comment:
The text said something about installing with values, but the code is the default installation. 
Maybe this order makes more sense...

```suggestion
. Install the latest version of the {productnameshort} {helm} chart.

[source]
$ helm install <RELEASE_NAME> \
oci://registry.suse.com/private-registry/private-registry-helm \
--namespace <PRIVATE_REGISTRY_NAMESPACE>

To override the default installation with custom values from the `suse_registry_override.yaml` file`, refer to <<pr-helm-chart-override>>.
```